packages/terra-clinical-item-display/src/terra-dev-site/doc/example/TruncatedText.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import ItemDisplay from 'terra-clinical-item-display';
+import Button from 'terra-button';
+
+const sampleText = 'Mr. James is currently receiving outpatient treatment. He has been diagnosed with an Axis I diagnosis of Psychosis NOS, ruled out Schizoaffective Disorder and Post Traumatic Stress Disorder, and is being treated with Haldol 5mg and Cogentin 1mg.';
+
+const TruncatedTextExample = () => {
+  const [truncated, setTruncated] = useState(true);
+
+  const handleClick = () => {
+    setTruncated(!truncated);
+  };
+
+  return (
+    <>
+      <ItemDisplay text={sampleText} isTruncated={truncated} />
+      <br />
+      <Button text="Show/Hide" onClick={handleClick} />
```

Comment:
Show/Hide component might be better here even though it has its own issues
https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/show-hide/about

Reply:
Updated [08908d6](https://github.com/cerner/terra-clinical/pull/848/commits/08908d6c569a6a09186c041b7eabfc9dab10bd86)